src/components/Editor/Editor.tsx

```diff
@@ -137,16 +155,14 @@ export const Editor: FunctionComponent<EditorProps> = props => {
                     <button onClick={onButtonClick} className={cx('preview')}>
                         {isPreview ? 'Edit' : 'Preview'}
                     </button>
-                    <a
+                    <Link
```

Comment:
Please, add TODO and a new ticket. It's not good to redirect a user at once, before file saving.